src/qt/font.cpp

```diff
@@ -469,9 +469,20 @@ void wxNativeFontInfo::SetPixelSize(const wxSize& size)
 
 void wxNativeFontInfo::SetStyle(wxFontStyle style)
 {
-    m_qtFont.setItalic(style == wxFONTSTYLE_ITALIC);
-//case wxFONTSTYLE_SLANT:
-//case wxFONTSTYLE_NORMAL:
+    switch(style)
+    {
+        case( wxFONTSTYLE_ITALIC ):
```

Comment:
Could you please remove the parentheses around the case labels? This style is not used anywhere else in wx AFAIK.